db/migrate/20230920044923_create_comment.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comment do |t|
+      t.text :body
+      t.references :user_id, null: false, foreign_key: { to_table: :user }
+      t.references :post_id, null: false, foreign_key: { to_table: :post }
+      # t.datetime :updated_at
+      # t.datetime :created_at
```

Comment:
- [ ] `user_id` create `user_id_id` in the `comments` table which I think is not great or best practice. Also best practice is to pluralized all table's name. Instead of `comment`, it should be `comments` You'll needs to fix this too in all the tables
- [ ] Please don't forget to remove comments